remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerInterceptor.java

```diff
@@ -199,7 +200,7 @@ private void registerProtoFile(String name, String content, FileDescriptorSource
 
    private void registerWithContextRegistry(FileDescriptorSource source) {
       try {
-         serializationContextRegistry.addProtoFile(SerializationContextRegistry.MarshallerType.GLOBAL, source);
+         ((SerializationContextRegistryImpl) serializationContextRegistry).getUserCtx().registerProtoFiles(source);
```

Comment:
I did it this way to quickly test whether the change would cause any issues. I agree it would be better to be consistent, I'll update the code to have `MarshallerType.USER`.